packages/e2e-tests/fixtures/blocks/core__button__squared.json

```diff
@@ -11,8 +11,7 @@
                     "text": "#1b9b6c",
                     "background": "#aa5a20"
                 }
-            },
-            "align": "none"
```

Comment:
Since the Button component uses the align hook, and the align hook now has the enum for accepted values (right, center, left, wide, full), the test for `core__button__squared` was failing due to not returning `"align": "none"` as the fixture expected.  I removed this attribute from the fixture files for this test.  Hopefully this makes sense? 🤔

Reply:
Or would it make more sense to leave this attribute in `core__button__squared.html` and `core__button__squared.parsed.json`, while only removing it from `core_button__squared.json` and `core__button__squared.serialized.html` (as opposed to just removing it from all 4)?  The idea being that if the attribute were declared in this way, we would expect it to not be returned according to the behavior of the align hook.

Reply:
My opinion is that it makes sense to remove from all 4. I think having it in some and not others could be confusing. Plus, we aren't directly trying to test the "align" hook here, so if we do want to test that behavior, shouldn't it go somewhere else?

I might be missing something here, though, let me know!